Code/GraphMol/FileParsers/MaeWriter.cpp

```diff
@@ -221,6 +223,61 @@ int bondTypeToOrder(const Bond& bond) {
   }
 }
 
+static bool is_double_any_bond(const RDKit::Bond& b)
+{
```

Comment:
wow, that's an annoying set of logic. It's a bummer that there are _four ways_ to encode this information